tests/openfl/utilities/test_change_tags.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""Data Splitters tests module."""
+import pytest
+
+from openfl.utilities import change_tags
+
+
+@pytest.mark.parametrize(
+    'tags,add_field,remove_field,expected_result', [
+        ('abc', None, 'abc', ()),
+        (('abc',), None, 'abc', ()),
+        (['abc'], None, 'abc', ()),
+        ('abc', 'def', None, ('abc', 'def')),
+        (['abc'], 'ghi', None, ('abc', 'ghi')),
+        (['abc', 'def'], 'ghi', None, ('abc', 'def', 'ghi')),
+        (['abc', 'def'], None, 'abc', ('def',)),
+        (['abc', 'def'], 'ghi', 'abc', ('def', 'ghi')),
+        (('abc', 'def'), 'ghi', 'abc', ('def', 'ghi')),
+        (('abc', 'def', 'def'), 'ghi', 'abc', ('def', 'ghi')),
+        (('abc', 'ghi', 'def'), 'ghi', 'abc', ('ghi', 'def')),
+    ])
```

Comment:
To fully test this function the following cases should be checked:
1. tags:
    a. list: empty, one element, two elements
    b. tuple: empty, one element, two elements
    c. string: empty, not empty
2. add_field: None, empty string, string that not in tags, string that in tags
3. remove_field: None, empty string, string that not in tags, string that in tags
4. add_field == remove_field: None, empty strings, string that not in tags, string that in tags

Better to create different tests for different cases. Think how to better organize groups for parametrize.

Reply:
@dmitryagapov in case remove_field value doesn't exist, we would throw an error instead. Let me know if we still want to add it to test.

Reply:
@mansishr, I think we should add test for this any way.

Reply:
Done